components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilder.java

```diff
@@ -276,7 +264,8 @@ public ServiceURL build() throws URLBuilderException {
             }
         }
 
-        return new ServiceURLImpl(protocol, hostName, port, resolvedUrlStringBuilder.toString(), parameters,
+        return new ServiceURLImpl(protocol, hostName, port, proxyContextPath, resolvedUrlStringBuilder.toString(),
+                parameters,
```

Comment:
formatting. Get the next line to this line

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/3063.